src/reporting/ReportingWindow.sol

```diff
@@ -67,7 +67,7 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable {
     function migrateMarketInFromNibling() public afterInitialized returns (bool) {
         Market _market = Market(msg.sender);
         Branch _shadyBranch = _market.getBranch();
-        require(branch.isParentOf(_shadyBranch));
+        require(_shadyBranch.isParentOf(branch));
```

Comment:
This check is insecure as `_shadyBranch` may lie about who its parent is.  If this is the direction we want, then this should be changed to `branch.getParentBranch() == _shadyBranch`.

Reply:
Ah good point. Yea I believe that will be what we have to do. The code before was just logically incorrect.